python/lsst/daf/butler/registry/queries/_query_backend.py

```diff
@@ -624,8 +623,7 @@ def resolve_governor_constraints(
 
         Returns
         -------
-        resolved : `Mapping` [ `str`,  [ `~collections.abc.Set`
-                [ `str` ] ] ]
+        resolved : `Mapping` [ `str`,  [ `~collections.abc.Set` [ `str` ] ] ]
```

Comment:
FTR, the [Python style guide](https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-dict) only requires spaces _before_ brackets, not inside them. The extra space adds up for highly nested types, and AFAIK it has no effect on the final HTML.

Reply:
I'm dreaming of a time when I don't need to put any of this in because the type annotations will be used directly.

Reply:
Personally, I hope that doesn't happen. Some of the broken links on #846 were the result of trying to use the exact same notation as the annotations, even when Sphinx couldn't understand them. In a few cases (e.g., "any collection expression"), the human-readable type is _much_ simpler than its machine-readable equivalent.

Reply:
While revisiting this line I realized that `resolved : mapping [ str,  [ set [ str ] ] ]` is not valid notation to begin with. Is this supposed to be a mapping from strings to sets, or a mapping from strings to lists of sets?